standalone/pruntime/enclave/src/lib.rs

```diff
@@ -57,7 +56,8 @@ pub extern "C" fn ecall_init(args: *const u8, args_len: usize) -> sgx_status_t {
         panic!("Enclave already initialized.");
     }
 
-    use parity_scale_codec::Decode;
+    libc_hacks::init();
```

Comment:
Where is it called before this PR?

Reply:
> Where is it called before this PR?

No where.
It was removed by mistake in [this commit](https://github.com/Phala-Network/phala-blockchain/commit/f6d611b09f7319dd5c9ece18f2fcd9f4a1080122#diff-37a68b9c61482f43c7d954f2cedf640b30d19bbfa40fe66589cc7e17ed99ccbaL51).